JS/Lesson35/onur/my-app/src/components/SignInPage/index.jsx

```diff
@@ -1,49 +1,69 @@
-import './styles.css'
-import { useForm } from 'react-hook-form';
+import { Box, Button, Stack, TextField, Typography } from "@mui/material";
+import { useForm } from "react-hook-form"
+import styled from "@emotion/styled";
+
+export const StyledBox = styled('div')(({ theme }) => ({
```

Comment:
Nice that you tried out styled components. Going forward, pick one way of doing styles for your project (css files, styles components, css-in-js) and try to follow it everywhere. It's always better to keep your code consistent.